sections/featured-collection.liquid

```diff
@@ -87,7 +88,7 @@
           </li>
         {%- else -%}
           {%- for i in (1..4) -%}
-            <li class="grid__item">
+            <li data-cascade class="grid__item{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
```

Comment:
I'm curious if there's any reason you opted to put `data-cascade` first? Do we need extra visibility on this attribute? 

Asking because we usually put `data-*` attributes last, after `id`, `class` and other element-specific attributes.